main.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"api/handlers"
+	"log"
+	"net/http"
+
+	"github.com/gorilla/mux"
+)
+
+const port = ":8080"
```

Comment:
We can get values like these from a configuration file instead. Take a look at [the viper package](https://github.com/spf13/viper).